work_plan.md

```diff
@@ -76,32 +72,42 @@ https://github.com/nilearn/nilearn/issues/2171.  We will develop a standard
 API, apply it to the three standard formats, and design a efficient general
 HDF5 storage container for serializing surface data and metadata.
 
-Range
-=====
-
-Registration transforms
------------------------
-
-Nibabel's role as a workbench allows researchers to combine outputs from
-processing in other packages.
-
-Image registration is an important step in neuroimaging.  Many packages
-estimate image registration parameters; each has their own format for storing
-these estimates.
-
-Using these estimates involves interpreting the stored transforms, and
-combining chains of transforms into a single transform, to allow efficient
-resampling of an image to match the position of another.
-
-We have developed a package, Nitransforms, that can read and process several
-of the stored parameter files.  We will integrate Nitransforms into Nibabel,
-expand tests and documentation, and saving in standard HDF5 format to store
-transforms, with labels implemented in Xarray and stored with HDF5.
-
-Strengthening social foundations
-================================
-
-We want to recruit more developers, from a larger, more diverse pool, and help
+## Range
```

Comment:
This will push us over the work limit by quite a bit.  Any way of moving some of this into the milestones?   We've plenty of space there.

Reply:
Sure thing